docs/samples/explanation/aif/germancredit/server/Dockerfile

```diff
@@ -13,7 +13,20 @@ RUN pip install --no-cache-dir -r ./requirements.txt
 COPY model.py ./
 
 # Copy german-credit data
-COPY german.data /usr/local/lib/python3.7/site-packages/aif360/data/raw/german/
-COPY german.doc /usr/local/lib/python3.7/site-packages/aif360/data/raw/german/
+COPY german.data /usr/local/lib/python3.10/site-packages/aif360/data/raw/german/
+COPY german.doc /usr/local/lib/python3.10/site-packages/aif360/data/raw/german/
 
 CMD ["python", "model.py"]
+# for local test use this CMD:
```

Comment:
This should be an accompanying README file instead of a block that’s commented out here

Reply:
make sense.
Done.